.github/workflows/release.yml

```diff
@@ -1,37 +1,36 @@
 name: release
 
 on:
-    push:
-        tags:
-            - 'v*'
+  push:
+    tags:
+      - "v*"
 
 permissions:
-    contents: write
+  contents: write
 
 jobs:
-    goreleaser:
-        runs-on: ubuntu-latest
-        steps:
-            - name: Checkout
-              uses: actions/checkout@v4
-            - name: Unshallow
-              run: git fetch --prune --unshallow
-            - name: Setup Go
-              uses: actions/setup-go@v5
-              with:
-                go-version: 1.22.4
-            - name: Import GPG Key
-              id: import_gpg
-              uses: crazy-max/ghaction-import-gpg@v6.2.0
-              with:
-                gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
-            - name: Run GoReleaser
-              uses: goreleaser/goreleaser-action@v6.0.0
-              with:
-                version: latest
-                args: release --clean
-              env:
-                GPG_FINGERPRINT: ${{ steps.import_gpg.outputs.fingerprint }}
-                # GitHub sets this automatically
-                GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Unshallow
+        run: git fetch --prune --unshallow
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.22"
+      - name: Import GPG Key
+        id: import_gpg
+        uses: crazy-max/ghaction-import-gpg@cb9bde2e2525e640591a934b1fd28eef1dcaf5e5
```

Comment:
We should comment the hashes with `# vX.X.X` like we do in `coder/coder`. Dependabot notices and bumps the comment too.